exporter.go

```diff
@@ -134,10 +135,12 @@ func (c *CephExporter) Collect(ch chan<- prometheus.Metric) {
 
 func main() {
 	var (
-		addr        = flag.String("telemetry.addr", ":9128", "host:port for ceph exporter")
-		metricsPath = flag.String("telemetry.path", "/metrics", "URL path for surfacing collected metrics")
-		cephConfig  = flag.String("ceph.config", "", "path to ceph config file")
-		cephUser    = flag.String("ceph.user", "admin", "Ceph user to connect to cluster.")
+		addr                  = flag.String("telemetry.addr", ":9128", "host:port for ceph exporter")
+		metricsPath           = flag.String("telemetry.path", "/metrics", "URL path for surfacing collected metrics")
+		cephConfig            = flag.String("ceph.config", "", "path to Ceph config file")
+		cephUser              = flag.String("ceph.user", "admin", "Ceph user to connect to cluster.")
+		cephRadosOSDOpTimeout = flag.Int64("ceph.rados_osd_op_timeout_sec", 30, "Ceph rados_osd_op_timeout used to contact cluster in seconds. 0 means on limit")
```

Comment:
We can just expose a single timeout option for rados ops and apply it to both to keep it simple.

Reply:
This field could also be `flag.Duration` instead of `flag.Int()`.

Reply:
👍